Fitness/Views/ProRoutineView.swift

```diff
@@ -0,0 +1,155 @@
+        
+        stepsTextView = UITextView()
+        stepsTextView.textColor = UIColor.fitnessBlack
+        stepsTextView.font = ._14MontserratLight
+        stepsTextView.isScrollEnabled = false
+        stepsTextView.textAlignment = .left
+        stepsTextView.isSelectable = false
+        stepsTextView.text = routine.text
+        stepsTextView.sizeToFit()
+        
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 5
+        let attributes = [NSAttributedString.Key.paragraphStyle : style, NSAttributedString.Key.font : UIFont._14MontserratLight]
+        stepsTextView.attributedText = NSAttributedString(string: routine.text, attributes: attributes)
+        
+        self.addSubview(stepsTextView)
+        
+        addRoundedCorners(radius: 5)
+        applySketchShadow(color: UIColor.fitnessGreen, alpha: 0.03, y: 11, blur: 14, spread: -10)
+        
+    }
+    
+    func addRoundedCorners(radius: CGFloat) {
+        layer.cornerRadius = radius
+        layer.masksToBounds = false
+        layer.borderColor = UIColor.fitnessMutedGreen.cgColor
+        layer.borderWidth = 1
+    }
+    
+    func applySketchShadow(color: UIColor = .black, alpha: Float = 0.5, x: CGFloat = 0, y: CGFloat = 2, blur: CGFloat = 4, spread: CGFloat = 0)
+    {
+        layer.shadowColor = color.cgColor
+        layer.shadowOpacity = alpha
+        layer.shadowOffset = CGSize(width: x, height: y)
+        layer.shadowRadius = blur / 2.0
+        if spread == 0 {
+            layer.shadowPath = nil
+        } else {
+            let dx = -spread
+            let rect = bounds.insetBy(dx: dx, dy: dx)
+            layer.shadowPath = UIBezierPath(rect: rect).cgPath
+        }
+    }
+    
+    override func updateConstraints() {
+        let iconRoutineLabelSpacing = 8
+        let iconSize = 15
+        let titleIconSpacing = 3
+        
+        titleLabel.snp.makeConstraints{ make in
+            make.leading.equalToSuperview().offset(ConstraintConstants.outerSpacing)
+            make.top.equalToSuperview().offset(ConstraintConstants.outerSpacing)
+            make.trailing.equalToSuperview().offset(-ConstraintConstants.outerSpacing)
+        }
+        
+        routineTypeImage.snp.makeConstraints{ make in
+            make.height.width.equalTo(iconSize)
+            make.top.equalTo(titleLabel.snp.bottom).offset(titleIconSpacing)
+            make.leading.equalTo(titleLabel)
+        }
+        
+        routineTypeLabel.snp.makeConstraints{ make in
+            make.leading.equalTo(routineTypeImage.snp.trailing).offset(iconRoutineLabelSpacing)
+            make.centerY.equalTo(routineTypeImage)
+        }
+        
+//        addButton.snp.makeConstraints{ make in
+//            make.height.width.equalTo(24)
+//            make.trailing.equalToSuperview().offset(-16)
+//            make.centerY.equalTo(titleLabel.snp.centerY)
+//        }
```

Comment:
delete commented out code

Reply:
That code is temporarily commented out until we have a way to display favourited routines and utilize the addButton